x-pack/plugins/enterprise_search/public/applications/shared/tables/reorderable_table/reorderable_table.test.tsx

```diff
@@ -5,6 +5,13 @@
  * 2.0.
  */
 
+/*
```

Comment:
nitpick: this should probably be removed, given that there's a license above here and there's an inadvertent `unreorderableItems` inserted into this block